.github/workflows/release.yml

```
@@ -86,8 +86,8 @@ jobs:
       - name: Add crates.io ownership
         if: ${{ steps.runtime_sub.outputs.is_shim != 'true' && github.repository == 'containerd/runwasi' && !inputs.initial_release }}
         run: |
-          cargo owner --list ${{ inputs.crate }} | grep github:containerd:runwasi-committers || \
-          cargo owner --add github:containerd:runwasi-committers ${{ inputs.crate }}
+          cargo owner --list ${{ inputs.crate }} | grep ${{ github.actor }} || \
```

I think that if runwasi-committers is an owner we don't need to add the GH actor
```suggestion
          cargo owner --list ${{ inputs.crate }} | grep -E 'github:containerd:runwasi-committers|${{ github.actor }}' || \
```